### PR TITLE
CSRF tags bulk lookup - Round 6

### PR DESCRIPTION
**Pull request by sravfeyn**
Bunch of fixes (2 are from logs, remaining all by find)
Needs submodules
https://github.com/dimagi/couchlog/pull/23
https://github.com/dimagi/ctable/pull/88

@TylerSheffels @czue

### CODE REVIEW

File: corehq/apps/performance_sms/templates/performance_sms/edit_performance_config.html [hunk 1]
```diff
@@ -22,7 +22,9 @@
     {% crispy form %}
     {% if editing %}
         <form method='post' action="{% url 'performance_sms.delete_performance_messages' domain form.config.get_id %}" >
+            {% csrf_token %}
             <input type="submit" value="{% trans 'Delete'%}" class="btn btn-danger disable-on-submit">
+            }
```
Comment (snopoke) [inline corehq/apps/performance_sms/templates/performance_sms/edit_performance_config.html]: stray `}`

**Comment by czue** [comment]
looks good once comments addressed

**Comment by czue** [comment]
(also not mergeable now)

**Comment by sravfeyn** [comment]
Updated (wait for tests though)

**Pull request merged**